dist/vue-gettext.js

```diff
@@ -841,6 +864,8 @@
 
     Config(Vue, languageVm, options.silent, options.autoAddKeyAttributes, options.muteLanguages);
 
+    translate.initTranslations(options.translations, Vue.config);
+
     // Makes <translate> available as a global component.
     Vue.component('translate', Component);
 
```

Comment:
Plase, [do not touch files in `dist`](https://github.com/Polyconseil/vue-gettext/blob/master/README_DEV.md#pull-request-guidelines) because they are automatically generated at release time.

Can you please remove them?